# object-oriented-programming
//write a c++ program to   use existing exception class to handle an exception for devide by 0
#include<iostream>
using namespace std;
class Myexception:public exception{
    public:const char*what()const throw(){
        cout<<"attempt devided by zero"<<endl;
    }
};
int main(){
    int a,b;
    cout<<"enter a & b  "<<endl;
    cin>>a>>b;
    try{
        if(b==0){
            throw Myexception();
        }
        cout<<a/b<<endl;
    }
    catch(Myexception e){
    e.what();
    }
}
